github/Organization.py

```diff
@@ -726,6 +760,19 @@ def has_in_public_members(self, public_member):
         )
         return status == 204
 
+    def remove_from_membership(self, member):
+        """
+        :calls: `DELETE /orgs/:org/memberships/:user <http://developer.github.com/v3/orgs/memberships>`_
```

Comment:
This url seems to point to nowhere

Reply:
hmm, likely copy/paste error on my part. fixed.